### PR TITLE
fixes UdaySravanK/RNSwipeButton#38

### LINKED ISSUE #38: EnableRightToLeftSwipe is not compatible with RTL Layout
I can't make the swipe gesture when I change the layout of the app to RTL layout for languages like Arabic language. it is not available, could you please help. If I kept it as default LeftToRight, I can't make the swipe as well in RTL layout.

### PR DESCRIPTION
**Pull request by mohamedmaged**
In this pull request I solved Fixes #38 
I have added a new prop called isRTL that has a default value to false 
`       ...(enableRightToLeftSwipe ? ( isRTL ? styles.container: styles.containerRTL ) : (isRTL ? styles.containerRTL : styles.container)),
`
by adding this extra check the RTL layouts will behave as expected

### CODE REVIEW

**Comment by SallyHabib97** [comment]
Finally Someone did that 
THANKS 🙈

**Comment by UdaySravanK** [comment]
Thanks for PR @mohamedmaged, I will take a look over the coming weekend.

File: src/components/SwipeThumb/index.js [hunk 1]
```diff
@@ -189,14 +189,15 @@ const SwipeThumb = props => {
...
+    isRTL
   } = props;
 
   const panStyle = {
     backgroundColor,
     borderColor,
     width: animatedWidth,
     ...railStyles,
-    ...(enableRightToLeftSwipe ? styles.containerRTL : styles.container),
+    ...(enableRightToLeftSwipe ? ( isRTL ? styles.container: styles.containerRTL ) : (isRTL ? styles.containerRTL : styles.container)),
```
Comment (UdaySravanK) [inline src/components/SwipeThumb/index.js:200]: Can you please try below things
1. Delete new prop 'isRTL' and try to use `I18nManager.isRTL`
2. Rename existing prop `enableRightToLeftSwipe` to `enableReverseSwipe`
3. By default for all RTL layouts swipe button should work as right to left swipe
Without any code changes, if we switch LTR to RTL then demo app should look like below screenshot

**Review by UdaySravanK — CHANGES_REQUESTED** [review]
Hey @mohamedmaged I don't think we need a new prop, please check my comment. thanks!

**Comment by mohamedmaged** [comment]
I think yes we don't need the isRTL prop , thanks for pinting that out , I will be doing the changes you reuqests in the cming weekend

**Comment by mohamedmaged** [comment]
@UdaySravanK  I did the changes you requested  I have made sure that By default for all RTL layouts swipe button should work as right to left swipe Without any code changes 

also removed the isRTL prop and used the I18nManager instead

File: docs/index.html [hunk 2]
```diff
@@ -117,6 +117,8 @@ <h2 style="color:darkgreen;">Component properties</h2>
     <b>titleStyles</b>: PropTypes.object,
 
     <b>width</b>: PropTypes.number,
+
+    <b>isRTL</b>: PropTypes.bool, <span style="color: blueviolet"> // for Right to left layout based Applications </span>
```
Comment (UdaySravanK) [inline docs/index.html:121]: Please delete this

**Review by UdaySravanK — CHANGES_REQUESTED** [review]
Hi Maged, Thanks for making changes. 
1. `enableReverseSwipe` not working. Can you please test and make necessary changes.
2. Please replace `enableRightToLeftSwipe` with `enableReverseSwipe` in `example/RNSwipeButtonDemo/App.js`

**Comment by UdaySravanK** [comment]
> Hi Maged, Thanks for making changes.
> 
>     1. `enableReverseSwipe` not working. Can you please test and make necessary changes.
> 
>     2. Please replace `enableRightToLeftSwipe` with `enableReverseSwipe` in `example/RNSwipeButtonDemo/App.js` also update 'Right to left swipe enabled' to 'Reverse swipe enabled'

Since the logic looks correct, I tried to test it multiple times. It is working well. I'm not sure why it was not working at first attempt. Please make changes for point 2 and you too test for one last time. Then we are good to go.

**Comment by mohamedmaged** [comment]
@UdaySravanK  I did the changes you requested , I have tested the app  one last time with and without RTL seems to be working fine

**Review by UdaySravanK — APPROVED** [review]
Appreciate your time. Nicely done 👍

**Comment by mohamedmaged** [comment]
@UdaySravanK  thanks :')

**Pull request merged**